flink-ml-lib/src/test/java/org/apache/flink/ml/util/StageTestUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.util;
+
+import org.apache.flink.ml.api.core.Stage;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+
+/** Utility methods for testing stages. */
+public class StageTestUtils {
+    /**
+     * Saves a stage to filesystem and reloads it with the static load() method a stage must
+     * implement.
+     */
+    public static <T extends Stage<T>> T saveAndReload(StreamExecutionEnvironment env, T stage)
+            throws Exception {
+        String tempDir = Files.createTempDirectory("").toString();
```

Comment:
OK. I'll do it.

The common practice in Flink tests is to initialize the `TemporaryFolder` in each test class, and calls for initialization in `@Before` methods. Since the usage of `TemporaryFolder` in our case is merely in static utility functions. I think we can just put it as a static member of that class and initializes in static context.